uo_sys_framework/Core/cfgFunctions/EndConditions/defines/Category1Loop.hpp

```diff
@@ -225,28 +225,35 @@ if (GETMVAR(Enabled_1,false)) then {
                 };
 
                 if (_ExtractionCheck) then {
-                    if (GVAR(Mode_1) isEqualto 1) then {
-                        {
-                            _x params ["_name","_value"];
-                            if (_value) exitwith {
-                                LOG_1("Category 1 Ending due to :%1",_value);
+                    if((GETMVAR(ExtractionEnabled_1,false)) && (GETMVAR(EndConditions,ExtractionForced_1,false))) then
```

Comment:
`GETMVAR(EndConditions,ExtractionForced_1,false)` should be `GETMVAR(ExtractionForced_1,false)`